### ISSUE TITLE
Implement themeing for text size and padding

### ISSUE THREAD

**Issue by thePlebDev** [thePlebDev/Clicker#904]
# Proposed change
- Add a compose theme for the text sizes and padding

# Why is this important
- This will allow me to give a easy and consistent feeling throughout my application
- Also, this will allow me to implement custom compose components easier

# Additional context
- n/a

**Comment by thePlebDev** [comment]
- after this is implemented I want to check if the `@stable`  annotation works for the anonymous objects. Like so:
```
@Stable
object Testing{
    
    @Composable
    fun testingRecomposition(){
        
    }
    
}

```
-  Use this code to see if recomposition is skipped

**Comment by thePlebDev** [comment]
- Also, I have to go through my composables to see which font size gets used the most

**Comment by thePlebDev** [comment]
I did the proof for: `after this is implemented I want to check if the @stable annotation works for the anonymous objects` in #905

**Comment by thePlebDev** [comment]
- closed with #906
